packages/component/src/BasicTranscript.tsx

```diff
@@ -200,27 +188,23 @@ const InternalTranscript = forwardRef<HTMLDivElement, InternalTranscriptProps>(
               showCallout = true;
             }
 
-            renderingElements.push({
-              activity,
-
-              // After the element is mounted, set it to activityElementsRef.
-              callbackRef: activityElement => {
-                activityElement
-                  ? activityElementMapRef.current.set(key, activityElement)
-                  : activityElementMapRef.current.delete(key);
-              },
-
-              // "hideTimestamp" is a render-time parameter for renderActivityStatus().
-              // If true, it will hide the timestamp, but it will continue to show the
-              // retry prompt. And show the screen reader version of the timestamp.
-              hideTimestamp:
-                hideAllTimestamps || indexWithinSenderAndStatusGroup !== entriesWithSameSenderAndStatus.length - 1,
-              key,
-              renderActivity,
-              renderActivityStatus,
-              renderAvatar,
-              showCallout
-            });
+            renderingElements.push(
+              <TranscriptActivity
+                activity={activity}
```

Comment:
I like that there are clear "setup-time props" and "render-time props" difference now. 😄